### PR TITLE
[APPLICATION] exam play game

### PR DESCRIPTION
**Pull request by darkfriend77**
Project Abstract
> Please replace these instructions with a brief description of your project summarising key points (1-2 paragraphs).

**Application Checklist**

[x] The [application template](https://github.com/PolkadotPlay/GamesBounty/blob/main/applications/application-template.md) or [delivery template](https://github.com/PolkadotPlay/GamesBounty/blob/main/deliveries/delivery-template.md) has been copied and aptly renamed for an application (`project_name.md`) and a delivery (`project_name_m1.md`).
[x] I have read the [games bounty guidelines](https://github.com/PolkadotPlay/GamesBounty/blob/main/docs/gamesbounty_guidelines.md).
[x] Payment details have been provided (Polkadot address in the application).
[x] The initial PR contains only one commit (squash and force-push if needed).

### CODE REVIEW

**Comment by johnrhodel** [comment]
Great

**Review by johnrhodel — DISMISSED** [review]

File: applications/exam-play-project.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+# EXAM PLAY PROJECT
+
+**NOT THIS IS A TEST APPLICATION, TO RUN THROUGH THE PROCCESS FOR THE CURATORS**
```
Comment (TychoMasius) [inline applications/exam-play-project.md]: Change to: 
**NOTE THIS IS A TEST APPLICATION, TO RUN THROUGH THE PROCESS FOR THE CURATORS**

**Review by TychoMasius — APPROVED** [review]

File: applications/exam-play-project.md (hunk 1 above)
Comment (TheWhiteRabbitM) [reply applications/exam-play-project.md]: Nice

**Review by TheWhiteRabbitM — APPROVED** [review]

File: applications/exam-play-project.md (hunk 1 above)
Comment (darkfriend77) [reply applications/exam-play-project.md]: Fixed the typo in the commit

**Review by TychoMasius — APPROVED** [review]

**Review by TychoMasius — APPROVED** [review]

File: applications/exam-play-project.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+- **If no,** *could you provide an excerpt or summary that highlights key aspects of your game's design, mechanics, and vision?*
+
+Yes, we do have a GDD, I can provide extracts of it if necessary, I will add partial Screenshots, of elements and their specification.
+
+## 2. Game Dev Experience
+
+### Have you built games prior to this bounty request?
+- **If yes,** *please list game name(s) & links:*
+
+Yes, Witchery (Unity), [Sabberstone](https://github.com/HearthSim/SabberStone) (Simulator), [The World of Mogwais](https://github.com/WorldOfMogwais/WoMNetCore/releases) (Web3 Game), [DOT4Gravity](https://github.com/ajuna-network/game-dot4-unity) (Web3 Game Layer 2), [AwesomeAjunaAvatars](https://aaa.ajuna.io/) (Web3 Game Layer 1), [Hexalem](https://github.com/SubstrateGaming/hackathon-2023-winter/tree/main/projects/36-Hexalem) (Demo Game), and BattleMogs (Web3 ongoing)
```
Comment (HopeClary) [inline applications/exam-play-project.md:90]: ✔️

**Review by HopeClary — DISMISSED** [review]
Love the name of the game. Great sample test, thank you!

**Review by TychoMasius — APPROVED** [review]

**Review by TychoMasius — APPROVED** [review]

**Review by TheWhiteRabbitM — APPROVED** [review]

**Review by TheWhiteRabbitM — APPROVED** [review]
awesome

**Review by Luca-Poggi — APPROVED** [review]
OK!

**Comment by HopeClary** [comment]
All good

**Comment by johnrhodel** [comment]
All good from my end

**Review by Luca-Poggi — COMMENTED** [review]
Done!

**Review by HopeClary — COMMENTED** [review]
Test

**Pull request merged**